src/gui/Application.cpp

```diff
@@ -324,46 +327,40 @@ void Application::socketReadyRead()
         return;
     }
 
-    QStringList fileNames;
-    quint32 id;
+    SocketCmd id;
     in >> id;
```

Comment:
What is the suggested way to solve this build failure in GCC? It compiles fine using clang. Should I change the type here back to `quint32` and only use the enum in the switch? Would I need to change it back in the sending side as well?

Reply:
Nevermind, this is due to the QT version. It should build correctly by default starting with 5.14. I added the fix from the later version directly